src/test/java/TestLIC0.java

```diff
@@ -0,0 +1,50 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestLIC0 {
+
+    @Test
+    public void testLIC0 () {
+        Decide decide = new Decide();
```

Comment:
Kom vi fram till något i `static` vs ej-`static` debatten? Tänker att om man kör `static` så slipper man skapa sådana här dummy objekt?

Reply:
I agree, we should use static.